packages/slate-commons/src/commands/blur.ts

```diff
@@ -0,0 +1,6 @@
+import type { Editor } from 'slate';
+import { ReactEditor } from 'slate-react';
+
+export function blur(editor: Editor & ReactEditor): void {
+    ReactEditor.deselect(editor);
```

Comment:
For some reason `ReactEditor.blur` doesn't do what I'd expect but `ReactEditor.deselect` does.

I wasn't sure about the function name but I think the `blur` is still correct as it does what is expected.